src/test/main.spec.test.coffee

```diff
@@ -53,6 +53,44 @@ describe 'slack new webhook test', ->
       response.headers.should.be.an.Object
       done()
 
+describe 'slack new Webhook class test', ->
+  this.timeout(50000)
+  webhook = new Slack.Webhook
+    url: webhookUri
+    channel: "#general"
+    username: "webhookbot"
+
+  it 'should create a webhook object', (done) ->
+    webhook.should.be.an.Object
```

Comment:
can you double check that this is actually an assertion? in previous libraries I've written, this has accidentally not asserted anything, and I think it depends on the version of `should` being used.

(if possible I'd rather avoid the uncertainty by doing something like `(typeof webhook).should.equal "object"`, or whatever assertion this is making under the covers)